linux/build.gradle.kts

```diff
@@ -15,5 +15,6 @@ dependencies {
      * will be pulled into plugin/build.gradle.kts when it specifies implementation(project(":auto-dark-mode-linux")).
      */
     implementation(project(":auto-dark-mode-linux-gnome"))
+    implementation(project(":auto-dark-mode-linux-xdg"))
```

Comment:
I tried to run the `runIde` task, but it fails because it can't find the `SigHandler` class (defined in `dbus-java`). I guess I have missed something in the gradle setup.

This is strange, because for my testing class I just added `testImplementation(projects.autoDarkModeLinuxXdg)` to the gradle setup of /plugin and everything works...

Reply:
Is this a runtime error or a build error? In the case of the former make sure you have run `gradlew build` before.

Reply:
It is a build error. Exact output is:
```
Supertypes of the following classes cannot be resolved. Please make sure you have the required dependencies in the classpath:
    class com.github.weisj.darkmode.platform.linux.xdg.XdgThemeMonitorService, unresolved supertypes: org.freedesktop.dbus.interfaces.DBusSigHandler
Adding -Xextended-compiler-checks argument might provide additional information.
```
I tried invalidating the caches and removed the `.gradle` directory to redownload everything, but the error remains.

Reply:
You’ll have to make `XdgThemeMonitorService` not extend `DBusSigHandler`, as it isn’t exported as api (which it shouldn’t). Instead implement `DBusSigHandler` through a separate class of which `XdgThemeMonitorService` holds an instance of.